src/app/containers/Image/index.jsx

```diff
@@ -20,6 +20,7 @@ const ImageContainer = ({ blocks }) => {
   const rawImageBlock = filterForBlockType(blocks, 'rawImage');
   const altTextBlock = filterForBlockType(blocks, 'altText');
   const captionBlock = filterForBlockType(blocks, 'caption');
+  const { copyrightHolder } = rawImageBlock.model;
```

Comment:
This should be destructured on [line 29](https://github.com/bbc/simorgh/pull/515/files#diff-445cfd09f5ef4bfbe3544be7b7f2da00R29)